test/podFailover/pod-failover-workload-chart/1pod1pvc.yaml

```diff
@@ -0,0 +1,23 @@
+storageClass:
```

Comment:
Consider renaming these files with storage type in the name? Or these are just templates?

Reply:
should just be generic-ish templates